tests/conftest.py

```diff
@@ -0,0 +1,167 @@
+import os
```

Comment:
@acarriedev can you review this as it was your suggestion?

Reply:
Looks great. Other that the default userID and asid which I think should be put into the tests as it's not clear in the tests where the value comes from and in relation to my comment on parametrizing